packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -241,7 +241,7 @@ export default class SourceEditing extends Plugin {
 		const data = {};
 
 		for ( const [ rootName, domSourceEditingElementWrapper ] of this._replacedRoots ) {
-			const oldData = editor.data.get( { rootName } );
+			const oldData = formatSource( editor.data.get( { rootName } ) );
```

Comment:
I missed it previously when I wrote it :sweat\_smile: But perhaps it would be good to store the roots data to not read them twice (`editor.data.get()` calls can be quite expensive). I'll create an issue for that.

Reply:
Good idea. You don't need to create a separate issue for that, I will add this simple improvement in this PR 🙂